src/components/node/sul-study-place/study-place-features.tsx

```diff
@@ -17,20 +17,26 @@ interface ModalProps {
   imagePlaceholder?: string
   type: string
   headingId?: string
+  roomNumber?: Maybe<string>
+  roomDonorName?: Maybe<string>
+  roomImageUrl?: string
+  roomImageAlt?: Maybe<string>
 }
 
-const StudyPlaceFeatures = ({branchHours, branchTitle, branchUrl, capacity, contactImageAlt, contactImageUrl, features, type, imagePlaceholder, headingId}: ModalProps) => {
+const StudyPlaceFeatures = ({branchHours, branchTitle, branchUrl, capacity, contactImageAlt, contactImageUrl, features, type, imagePlaceholder, headingId, roomNumber, roomDonorName, roomImageUrl, roomImageAlt}: ModalProps) => {
+  const ImageUrl = roomImageUrl || contactImageUrl;
+  const ImageAlt = roomImageAlt || contactImageAlt
```

Comment:
I normally use lower case for the first character unless it's a React component.